tools/diagnose.py

```diff
@@ -96,13 +96,8 @@ def check_mxnet():
         print('Version      :', mxnet.__version__)
         mx_dir = os.path.dirname(mxnet.__file__)
         print('Directory    :', mx_dir)
-        commit_hash = os.path.join(mx_dir, 'COMMIT_HASH')
-        if os.path.exists(commit_hash):
-            with open(commit_hash, 'r') as f:
-                ch = f.read().strip()
-                print('Commit Hash   :', ch)
-        else:
-            print('Commit hash file "{}" not found. Not installed from pre-built package or built from source.'.format(commit_hash))
+        print('Branch       :', mxnet.base.get_branch())
+        print('Commit Hash  :', mxnet.base.get_commit_hash())
```

Comment:
Our website points people to this script in master branch regardless of the version, and as a result some older builds would not have these interfaces. It would be better to do a try/except here.

Reply:
Done